releasenotes/notes/option_ignore_kube_service-e3d7864fbe726e3a.yaml

```diff
@@ -0,0 +1,25 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add an option and an annotation to skip ``kube_service`` tags on Kubernetes pods.
+
+    When the selector of a service matches a pod and that pod is ready, its metrics are decorated with a ``kube_service`` tag.
+
+    When the readiness of pod flips, so does the ``kube_service`` tag. This could create visual artifacts (spikes when the tag flips) on dashboards where the queries are missing ``.fill(null)``.
```

Comment:
```suggestion
    When the readiness of a pod flips, so does the ``kube_service`` tag. This could create visual artifacts (spikes when the tag flips) on dashboards where the queries are missing ``.fill(null)``.
```